modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -105,8 +105,7 @@ same data center as the Runtime Manager app.
 == See Also
 
 * https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/object-store-v2/[Anypoint Exchange Object Store v2]
-* https://www.anypoint.mulesoft.com/exchange/org.mule.modules/mule-module-objectstore/[Mule 3 Object Store Connector]
 * https://www.anypoint.mulesoft.com/exchange/org.mule.connectors/mule-objectstore-connector/[Mule 4 Object Store Connector]
-* xref:index.adoc[Object Store Documentation]
+* https://www.anypoint.mulesoft.com/exchange/org.mule.modules/mule-module-objectstore/[Mule 3 Object Store Connector]
```

Comment:
Should be "Object Store Connector - Mule 3" to be consistent to other usages in these docs.